.github/workflows/ci-pipeline-arm64.yml

```diff
@@ -0,0 +1,182 @@
+name: Arm64 CI
+
+# github.repository has been used to ensure CI is only run on the repo where
+# self-hosted runners are installed. This will prevent Arm64 CI failing on forks
+
+on: [push, pull_request]
+env:
+  ARCH: arm64
+  CC: gcc
+  CONTAINER_NAMESPACE: ghcr.io/opendataplane/odp-docker-images
+  OS: ubuntu_20.04
+
+jobs:
+
+  Build:
+    runs-on: self-hosted
```

Comment:
One could potentially have runners with multiple architectures, so the following would be better:
`runs-on: [self-hosted, ARM64]`

Same comment for all `runs-on` lines in this file.